client/src/components/app.jsx

```diff
@@ -30,20 +28,10 @@ class App extends React.Component {
   }
 
   nextListing(event) {
-    let url = window.location.href.split('/');
-    let id = parseInt(url[url.length - 2]);
-
-    if (id === 100) {
-      id = 1;
-      url[url.length - 2] = id;
-    } else {
-      id++;
-      url[url.length - 2] = id;
-    }
-    let prevListing = url.join('/')
-    console.log('New for URL :', prevListing)
-    window.location.assign(prevListing)
-    axios.get(prevListing)
+    let url = window.location.pathname.split('/').filter(n => n);
+    url[1]= Number(url[1]) + 1;
```

Comment:
What happens if you are on homesdata for the very last listing in the database and you hit 'nextListing'?  
Also, this might be an edge case, and I guess it depends on how the app and database are set up, but if a listing is deleted, would this break?

Reply:
This is legacy code, the front-end doesn't have any way of hitting next or previous listing, the functions are just there, unused.